features/user/components/userProfile/friendsPanel/FriendsList.tsx

```diff
@@ -0,0 +1,96 @@
+'use client'
+
+import React, { useEffect, useState } from 'react'
+
+import { getUserById, getUsers } from '@/features/user/actions'
+import { User } from '@/features/user/types/User'
+import { FriendsListItem } from './FriendsListItem'
+import { LoadingSpinner } from '@/components/ui/LoadingSpinner'
+import { Invitation } from '@prisma/client'
+import { PaginationBar, Searchbar } from '@/shared'
+import { Search } from 'lucide-react'
+
+interface FriendsListProps {
+	currentUserId: string
+	requests?: Invitation[]
+}
+
+export const FriendsList = ({ currentUserId }: FriendsListProps) => {
```

Comment:
ogólnie to nie wiem czy w tych komponentach jest potrzebne przekazywanie currentUserId w propsie. Jeżeli chodzi o aktualnie zalogowanego user'a to chyba możesz po prostu w tym komponencie z sesji wyciągnąć